internal/checker/resolve_script.go

```diff
@@ -27,12 +27,12 @@ func (s SpitoRulesYaml) getRulesStructVal(key string) (RuleConf, bool) {
 	return val, ok
 }
 
-func getRulePath(ruleSetLocation RuleSetLocation, ruleName string) (string, error) {
+func getRulePath(rulesetPath, ruleName string) (string, error) {
 	// Support for both .yaml and .yml
-	spitoRulesDataBytes, err := os.ReadFile(ruleSetLocation.GetRuleSetPath() + "/spito-rules.yaml")
+	spitoRulesDataBytes, err := os.ReadFile(rulesetPath + "/spito-rules.yaml")
 	if errors.Is(err, fs.ErrNotExist) {
 		var _err error
-		spitoRulesDataBytes, _err = os.ReadFile(ruleSetLocation.GetRuleSetPath() + "/spito-rules.yml")
+		spitoRulesDataBytes, _err = os.ReadFile(rulesetPath + "/spito-rules.yml")
```

Comment:
I think we should use our `ConfigFilename` constant here instead of "/spito-rules.yml"

Reply:
It isn't a good idea, because we should handle not only `.yml` but also `.yaml`
I think that `ConfigFilename` should contain `spito-rules` instead of spito-rules.yml`